### DISCUSSION TITLE
Age 0 for survey comps

### DISCUSSION THREAD

**Discussion by brianlangseth-NOAA** [pfmc-assessments/canary_2023#57] [Q&A]
@okenk What did you decide regarding age 0 comps for the surveys? I was glancing at the lingcod comps and notice age 0 is included there in the age bins. We could do likewise and capture that information in the comps since there is data there.

**Comment by okenk** [comment]
There are only 6 age 0 fish, 248 age 1, 528 age 2. I think the default behavior is that the age 1 bin includes all fish ages 1 and younger. That is probably the way to go.

**Reply by brianlangseth-NOAA** [reply]
With that sample size I am very fine with that. You are right, the age=1 bin will cover anything lower. I didn't realize the sample size was that low. Maybe I should add that context to my email out to the survey group.
